spec/graphql/bamboohr_spec.rb

```diff
@@ -1,21 +1,25 @@
 require 'rails_helper'
 require 'bamboozled'
 
+TODAY_LONG = Time.zone.today.to_formatted_s(:iso8601)
+TODAY_SHORT = Time.zone.now.strftime("%m-%d")
+DATES = [TODAY_LONG].freeze
```

Comment:
Lets not shift the time period for both birthdays and today. This sets us up to have flaky tests. We can just set this to be the date you first created the test when it was working and leave the json data as static.
`DATES = ["2021-07-22"].freeze`